package.json

```diff
@@ -16,10 +16,12 @@
     "format:check": "npm run prettier && npm run lint",
     "prepare": "husky",
     "docker:infrastructure": "docker compose -f docker/infrastructure.yml build && docker stack deploy -c docker/infrastructure.yml rcri",
-    "docker:build-dev": "TAG=dev PROJECT_DOCKERFILE=Dockerfile.dev docker compose -f docker/services.build.yml build",
-    "docker:services-dev": "TAG=dev docker stack deploy -c docker/services.dev.yml rcrs",
-    "docker:build": "PROJECT_DOCKERFILE=Dockerfile docker-compose -f docker/services.build.yml build",
-    "docker:services": "docker stack deploy -c docker/services.yml rcrs",
+    "docker:build-dev": "docker build -t rod_catch_returns/api_js_dev --target development .",
+    "docker:service-dev": "docker run -d --rm --env-file .env -p 5000:5000 -v ${PWD}/:/app --name rcr_api_js_dev rod_catch_returns/api_js_dev",
+    "docker:stop-dev": "docker container kill rcr_api_js_dev",
+    "docker:build": "docker build -t rod_catch_returns/api_js --target production .",
+    "docker:service": "docker run -d --rm --env-file .env -p 5000:5000 --name rcr_api_js rod_catch_returns/api_js",
+    "docker:stop": "docker container kill rcr_api_js",
```

Comment:
Great that you've added these additional commands, but I think some use cases in the README might be handy

Reply:
Good point, I've updated the README